adagucserverEC/CRequest.cpp

```diff
@@ -2174,7 +2174,13 @@ int CRequest::process_querystring(){
     CDBDebug("pszPATH %s = %s",key,getenv(key));
   }*/
   
-  
+
+  #ifdef ADAGUC_USE_KDCMONGODB
```

Comment:
Mogelijk kan setConfig een statische methode worden die de global *configurationObject aanpast. Dan kan de new () in elk geval weg.